src/coprocessor/tracker.rs

```diff
@@ -67,6 +67,7 @@ pub struct Tracker {
     total_perf_stats: PerfStatisticsDelta, // Accumulated perf statistics
     slow_log_threshold: Duration,
 
+    read_used_time: i64,
```

Comment:
I'm also not good at naming things. cc @breeswish Is `used` proper, or `spent`?

Also, what's your consideration about the type of `i64`? Will `Duration`, at least `u64`, be more accurate?